CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,194 @@
+* Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory
+  jokes and language.
+* Posting or displaying sexually explicit or violent material.
+* Posting or threatening to post other people's personally identifying
+  information ("doxing").
+* Personal insults, particularly those related to gender, sexual or romantic
+  orientation, ethnicity, religion or disability.
+* Inappropriate photography or recording. Note that photography or recording
+  without prior consent is _always_ inappropriate.
+* Inappropriate physical contact. You must have someone's consent before
+  touching them.
+* Unwelcome sexual attention. This includes, but is not limited to: sexualized
+  comments or jokes, inappropriate touching, groping and unwelcome sexual
+  advances. If you do not have someone's prior consent, then any kind of sexual
+  attention is unwelcome.
+* Deliberate intimidation or stalking (online or in person).
+* Sustained disruption of community spaces or events. This includes brining
+  weaponry into community spaces; see below for a more precise definition.
+* Intentional violation of local laws or regulations at in-person events.
+* Advocating for, encouraging, or assisting in any of the above behaviour.
+
+'Weapons' include, but are not limited to, guns, explosives (including 
+fireworks without prior approval by a community organizer), and large knives 
+such as those used for hunting or display, as well as any other item used for 
+the purpose of causing injury and harm to others. 
+
+## Consequences of unacceptable behaviour
+
+Unacceptable behaviour from any community member, including anyone with
+decision-making authority, will not be tolerated. Anyone asked to stop
+unacceptable behaviour is expected to comply immediately.
+
+If a community member engages in unacceptable behaviour, the community
+organizers may take any action they deem appropriate, subject to correction 
+and reversal by the lead organizer. These include, but are not limited to:
+
+* A verbal warning (either in private or in public).
+* Eviction from an event with no refund (if applicable), with conditional 
+  right of return or without right of return.
+* A temporary ban of any length, with or without right of appeal, from any or
+  all community spaces.
+* A permanent ban, with or without right of appeal, from any or all community
+  spaces.
+* Temporary revocation of community organizer status for any length of time, 
+  with or without right of appeal.
+* Permanent revocation of community organizer status with or without right of 
+  appeal.
+* Removal of contributions to Pext from any Pext-associated repositories, with
+  or without right of appeal.
+* Notification of law enforcement, or other appropriate authorities, along with
+  any evidence of wrongdoing provided to the community organizers.
+
+## Reporting guidelines
+
+If you are subject to, or witness, unacceptable behaviour, or have any other
```

Comment:
Should be "or witness of,".

Reply:
I just want to point out that witness is also a verb, so "if you witness unacceptable behaviour" could be valid.
Sorry for hijacking :)